src/network_filter.cc

```diff
@@ -155,4 +155,3 @@ NLV_WORKER_EXECUTE(NetworkFilter, ToXml)
 }
 
 }   // namespce NodeLibvirt
-
```

Comment:
removing every final line :) this is usually a requirement, I'm surprised github isn't complaining about it.